### PR TITLE
PROPOSAL: Name changes

### PR DESCRIPTION
**Pull request by IgaBroton**
I think some names could be confused so I tried to change it a little bit

### CODE REVIEW

File: league/manager/models.py [hunk 1]
```diff
@@ -46,33 +48,37 @@ def league_winner(self):
 
 
 class Team(models.Model):
-    league = models.ForeignKey(League, on_delete=models.CASCADE)
+    team_league = models.ForeignKey(League, on_delete=models.CASCADE)
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:51]: What about teams_league? I think that it would be easier to understand.

File: league/manager/models.py [hunk 2]
```diff
@@ -1,40 +1,42 @@
+from enum import Enum
 from django.core.exceptions import ValidationError
 from django.db import models
-from datetime import datetime
+from django.utils import timezone
 
 
 class League(models.Model):
-    name = models.CharField(max_length=50)
+    league_name = models.CharField(max_length=50)
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:8]: I don't think it was that necessary here, it's pretty clear that it meant the name of the league.

File: league/manager/models.py [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
     points_for_win = models.IntegerField(default=3)
     points_for_lost = models.IntegerField(default=0)
     points_for_draw = models.IntegerField(default=1)
-    max_number_of_teams = models.IntegerField(default=10)
+    max_number_of_teams_in_league = models.IntegerField(default=10)
+    max_number_of_players_in_team = models.IntegerField(default=2)
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:13]: I like this one, that's a good one.

File: league/manager/models.py [hunk 4]
```diff
@@ -1,40 +1,42 @@
...
 
     def __str__(self):
-        return self.name
+        return self.league_name
 
     @property
-    def teams_number(self):
+    def number_of_teams_in_league(self):
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:19]: That's great one, actually.

File: league/manager/models.py [hunk 5]
```diff
@@ -1,40 +1,42 @@
...
         return self.team_set.all().count()
 
     @property
-    def all_matches(self):
+    def number_of_all_matches_in_league(self):
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:23]: This one also helps readability of the code.

File: league/manager/models.py [hunk 6]
```diff
@@ -1,40 +1,42 @@
...
-    def played_matches(self):
-        return self.match_set.filter(winner__isnull=False, drawn__isnull=False).count()
+    def number_of_played_matches_in_league(self):
+        return self.match_set.filter(event__isnull=False).count()
 
     @property
-    def is_ended(self):
-        if self.all_matches > 0:
-            return self.all_matches == self.played_matches
+    def league_is_ended(self):
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:31]: This one is good.

File: league/manager/models.py [hunk 7]
```diff
@@ -46,33 +48,37 @@ def league_winner(self):
...
-        return self.number_of_matches_won * self.league.points_for_win + \
-               self.number_of_matches_drawn * self.league.points_for_draw + \
-               self.number_of_matches_lost * self.league.points_for_lost
+        return self.number_of_matches_won * self.team_league.points_for_win + \
+               self.number_of_matches_drawn * self.team_league.points_for_draw + \
+               self.number_of_matches_lost * self.team_league.points_for_lost
 
     @property  # TODO:Zuzannka77 add test to check if this property works properly
-    def matches_team_played(self):
+    def number_of_played_matches_by_team(self):
```
Comment (elzbietagawickaLOVE) [inline league/manager/models.py:66]: I mean, it's for sure clearer to understand.

File: league/manager/models.py (hunk 1 above)
Comment (Shefour) [reply league/manager/models.py:51]: team_league is already fine enough

**Pull request merged**